cassdegrees/templates/list.html

```diff
@@ -21,7 +21,7 @@
         <p class="msg-error">{{ render.error|linebreaksbr }}</p>
     {% endif %}
 
-    <form class="box bdr-solid bdr-uni bg-uni50 anuform custom_search_bar" action="?action=Search" method="get">
+    <form class="box bdr-solid bdr-uni bg-uni50 anuform custom_search_bar" action="/list/?action=Search" method="get">
```

Comment:
Should this be relative?

Reply:
I'm not sure where this has come from, I don't have in on my local as a change that I have made?

Reply:
Matched to develop